docs/source/format/CanonicalExtensions.rst

```diff
@@ -72,4 +72,57 @@ same rules as laid out above, and provide backwards compatibility guarantees.
 Official List
 =============
 
-No canonical extension types have been standardized yet.
+Fixed shape tensor
+==================
+
+* Extension name: `arrow.fixed_shape_tensor`.
+
+* The storage type of the extension: ``FixedSizeList`` where:
+
+  * **value_type** is the data type of individual tensors and
+    is an instance of ``pyarrow.DataType`` or ``pyarrow.Field``.
+  * **list_size** is the product of all the elements in tensor shape.
+
+* Extension type parameters:
+
+  * **value_type** = Arrow DataType of the tensor elements
+  * **shape** = shape of the contained tensors as a tuple
```

Comment:
`dim_names` and `permutation` are also parameters (optional). So maybe we could move the actual description of the parameters here, and below in the "Description of the serialization" bullet point only focus on the formatting of the parameters in the serialized metadata?

Reply:
OK, will do - I misunderstood and thought we were only talking about defining the metadata for this cases, not adding them to parameters.

Reply:
There is not a strict difference between both. The parameters are the information a user would need to provide to fully define an instance of this extension type (in in a C++ or Python API), and so the optional dim_names/permutation is part of that (even though they are not essential for the interpretation, hence they are optional). 
_Most_ of the parameters are then represented in the serizalization metadata (but `value_type` is not, because that parameter is already part of the physical storage type).